cloudify_starlingx/decorators.py

```diff
@@ -57,5 +66,8 @@ def wrapper_inner(**kwargs):
                     'Failure while trying to run operation:'
                     '{0}: {1}'.format(ctx.operation.name, message),
                     causes=[exception_to_error_cause(errors, tb)])
+            if cafile:
```

Comment:
I think the removal should always happen, consider putting it inside finally block.

Reply:
But it does not always exist.

Reply:
ok I mean you can put the whole if inside finally because if an exception rised in 54 you don't delete the file if it exists.